product_docs/docs/biganimal/release/getting_started/identity_provider/index.mdx

```diff
@@ -78,17 +78,50 @@ Once your identity provider is set up, you can view your connection status, ID,
 You need a verified domain so your users can have a streamlined login experience with their email address. 
 
 1. On the **Domains** tab, enter the domain name and select **Next: Verify Domain**. 
-1. Copy the TXT record and follow the instructions. 
+1. Copy the TXT record and follow the instructions to add it as a TXT record on that domain within your DNS provider's management console.
```

Comment:
Yes - the user's have to add the TXT record to their domain in their DNS provider's system. So usually a website. This will be the same in nature, but differ in details, for adding / editing a domain TXT record.

The instructions referred to before in this help doc were just the generic ones that appear within the verify box of the UI.

I added more detailed generic instructions in my support doc for Domains here ... https://enterprisedb.atlassian.net/wiki/spaces/PD/pages/3866034269/Domain+Support#Customer-edit-DNS-records

Do you think these should be added to the Customer doc?

Reply:
I have updated the original part of the sentence to explain which instructions. The on screen instructions are as follows:

Instructions
Here's how you might enter that information into the DNS settings for your domain:
1. Log in to your domain registrar or web host account.
2. Navigate to the DNS settings for the domain you wish to verify.
3. Add a new TXT record.
4. In the "Name" field, enter "@" (without the quotes).
5. In the "Value" field, enter the string provided above.
6. Save your changes and wait for the DNS propagation period to complete (this may take up to 48 hours).

Reply:
Are you going to add these instructions? If so, here they are, edited:

1. Log in to your domain registrar or web host account.
1. Navigate to the DNS settings for the domain you want to verify.
1. Add a TXT record.
1. In the **Name** field, enter `@`.
1. In the **Value** field, enter the string provided above.
1. Save your changes and wait for the DNS propagation period to complete. This can take up to 48 hours.

But for the string, I'm not sure which string you are talking about. Better not to refer to things above and below in text but to call out more explicitly what you're referring to or use a link to the section where it's discussed.

Reply:
OK thanks, I have added the UI text edited as above (it was created by the doc or UI team btw)  and changed string to be the same example as in the nslookup.